packages/url_launcher/url_launcher_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.18
+
+* Removes nested third_party safari check.
```

Comment:
One last thing:

```suggestion
* Removes nested third_party Safari check.
```

Reply:
Also, please run `dart format .` within the `url_launcher_web`, the **`repo_checks` check is failing** because there's bad Dart formatting.